release.xml

```diff
@@ -69,6 +69,7 @@
             <zipfileset dir="${ts.home}/docs/"
                         includes="README.md"
                         prefix="mail-tck"/>
+        	<zipfileset dir="${ts.home}" includes="QuickStart.txt" prefix="mail-tck/docs/html-usersguide"/>
```

Comment:
use spaces instead of tab next time, please

Reply:
Done. My Eclipse IDE only uses 4 spaces in java code, I need to find how to set that in any other type of files.